lib/invoker/power/url_rewriter.rb

```diff
@@ -27,6 +28,12 @@ def extract_host_from_domain(complete_path)
 
       private
 
+      def dev_match_regex
+        # [/([\w.-]+)\.dev(\:\d+)?$/, /([\w-]+)\.dev(\:\d+)?$/]
+        tld = Invoker.tld
+        [/([\w.-]+)\.#{tld}(\:\d+)?$/, /([\w-]+)\.#{tld}(\:\d+)?$/]
```

Comment:
What happens when user configures tld as `.development` vs `development`, for example? is there any check preventing them to check for the starting `.`?